pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -0,0 +1,88 @@
+package component
+
+import (
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/golang/glog"
+
+	adapterCommon "github.com/openshift/odo/pkg/devfile/adapters/common"
```

Comment:
this can just be `"github.com/openshift/odo/pkg/devfile/adapters/common"` so that it matches with kubernetes/adapter.go

Reply:
It can't be because of the `github.com/openshift/odo/pkg/devfile/versions/common` import.